diff --git a/node_modules/react-native-wechat-lib/android/src/main/java/com/theweflex/react/WeChatModule.java b/node_modules/react-native-wechat-lib/android/src/main/java/com/theweflex/react/WeChatModule.java
index c563522..121434c 100644
--- a/node_modules/react-native-wechat-lib/android/src/main/java/com/theweflex/react/WeChatModule.java
+++ b/node_modules/react-native-wechat-lib/android/src/main/java/com/theweflex/react/WeChatModule.java
@@ -111,7 +111,7 @@ public class WeChatModule extends ReactContextBaseJavaModule implements IWXAPIEv
 
     @Override
     public String getName() {
-        return "RCTWeChat";
+        return "WeChat";
     }
 
     /**
